### ISSUE TITLE
ULMFIT PT e ES

### ISSUE THREAD

**Issue by lpcinelli** [mesquita/meli#1]
Nossa primeira tentativa deveria ser treinar um umlfit simples para cada linguagem (de repente pega ja pretreinado mesmo, dá para achar) e baseado no label da língua decide para qual modelo passar. Imagino que isso deve ser pelo menos tão bom quanto o baseline de 0.8755. Também acho que isso não deva nos colocar entre os 5 primeiros pois é muito simples e já devem ter tentado isso.

## ULMFIT PT

[Discussão no forum fast.ai](https://forums.fast.ai/t/ulmfit-portuguese/29366) :
> We and @NandoBr worked in parallel with relation to Portuguese ULMFit LM. I posted some results in my blog: https://monilouise.wordpress.com/2018/08/14/general-domain-portuguese-language-model/ 145. I believe the perplexity results may improve with further training epochs.
> 
> Further, I used my pretrained language model to test it for classification purposes with a Portuguese dataset (prepared by @NandoBr) and the best accuracy obtained was 97%. @NandoBr has also carried out similar experiments and may share its results. AFAIK, his language model also had excellent results in perplexity. Here’s the source code: https://github.com/fastai-bsb/nlp-tcu-enunciados/blob/master/1cycle_tcu.ipynb 96.

[Github da autora do post](https://github.com/monilouise/deep-learning/tree/master/nlp/language-models/pt)

## ULMFIT ES

[Discussão no forum fast.ai](https://forums.fast.ai/t/ulmfit-spanish/29715). Nesse link além da discussão, logo no começo tem uma tabela com links relevantes que valem a pena ser vistos, dentre eles essas duas implementações diferentes: 
- https://github.com/fpingham/SpanishULMFit/blob/master/TASS/Spanish-GeneralTASS.ipynb
- https://github.com/adai183/ULMFiT_es
